tests/test_calculator.py

```diff
@@ -64,9 +64,8 @@ def test_evaluate_success(fake_logger, fake_evaluator, fake_algorithm, sessions,
     results = calc.evaluate(
         algorithms=[fake_algorithm],
         evaluator_function=fake_evaluator,
-        sessions=sessions,
-        reference_session=reference_session,
-        count_of_sessions_to_evaluate=2
+        sessions=sessions[:1],
```

Comment:
this evaluates 1 session, not 2

<img width="119" height="82" alt="Image" src="https://github.com/user-attachments/assets/2be6ad13-2a26-4d13-88b4-e14bfd207978" />